### PR TITLE
VersionString() API selftest

### PR DESCRIPTION
**Pull request by grantseltzer**
**FAQ**

Is this the most useful selftest? No. 

Is it good to have? ok.

Signed-off-by: grantseltzer <grantseltzer@gmail.com>

### CODE REVIEW

**Review by kakkoyun — APPROVED** [review]

File: selftest/version/main.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package main
+
+import "C"
```
Comment (rafaeldtinoco) [inline selftest/version/main.go:3]: we dont need import C here.

File: selftest/version/main.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	cmd.Dir = "../../libbpf"
+
+	b, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// libbpf doesn't put the patch version in exported version
+	// symbols, so use just prefix to exclude it
+	if strings.HasPrefix(libbpfgo.VersionString(), string(b)) {
```
Comment (rafaeldtinoco) [inline selftest/version/main.go:25]: When I first read libbpfgo.VersionString() I thought we were comparing libbpf version with libbpfgo version (to see if libbpf string was contained in libbpfgo string). Should we change "VersionString" name ?

**Review by rafaeldtinoco — DISMISSED** [review]
LGTM but I would remove the unneeded import and think about changing the version function name (it looks like we are giving libbpfgo version string and not libbpf one).

File: selftest/version/main.go (hunk 2 above)
Comment (grantseltzer) [reply selftest/version/main.go:25]: Sure!

**Comment by grantseltzer** [comment]
> 

Done!

**Review by rafaeldtinoco — APPROVED** [review]
LGTM

**Pull request merged**